st2common/st2common/policies/meta/concurrency.yaml

```diff
@@ -6,5 +6,13 @@ resource_type: action
 module: st2actions.policies.concurrency
 parameters:
     threshold:
+        description: Concurrency threshold.
         type: integer
         required: true
+    action:
```

Comment:
This actually might end up confusing people because ``action`` is a content type in st2 world. I am trying hard to come up with an alternate but I can't.